konnectd/CHANGELOG.md

```diff
@@ -0,0 +1,151 @@
+# Changelog for [0.3.2] (2020-07-23)
+
+The following sections list the changes in ocis-konnectd 0.3.2.
+
+[0.3.2]: https://github.com/owncloud/ocis/konnectd/compare/v0.3.1...v0.3.2
+
+## Summary
+
+* Bugfix - Add silent redirect url: [#69](https://github.com/owncloud/ocis/konnectd/issues/69)
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#71](https://github.com/owncloud/ocis/konnectd/pull/71)
+
+## Details
+
+* Bugfix - Add silent redirect url: [#69](https://github.com/owncloud/ocis/konnectd/issues/69)
+
+   Adds the oidc-silent-redirect.html phoenix uses to silently refresh access tokens to the
+   default identifier-registration.yml
+
+   https://github.com/owncloud/ocis/konnectd/issues/69
+   https://github.com/owncloud/ocis/konnectd/pull/70
+
+
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#71](https://github.com/owncloud/ocis/konnectd/pull/71)
+
+   ARM builds were failing when built on alpine:edge, so we switched to alpine:latest instead.
+
+   https://github.com/owncloud/ocis/konnectd/pull/71
+
+# Changelog for [0.3.1] (2020-04-14)
+
+The following sections list the changes in ocis-konnectd 0.3.1.
+
+[0.3.1]: https://github.com/owncloud/ocis/konnectd/compare/v0.3.0...v0.3.1
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `142`)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)